src/main/archetype/ui.apps/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Created by https://www.gitignore.io/api/eclipse,java,maven
+
+src/main/content/jcr_root/apps/${appsFolderName}/clientlibs/clientlib-site
```

Comment:
@bpauli - these were removed in https://github.com/adobe/aem-core-wcm-components/issues/866 for the default (no ui-frontend module enabled), should we conditionally add this?

Reply:
@richardhand I think you wanted to reference to https://github.com/adobe/aem-project-archetype/issues/278, right?

This was my intention by delete the gitignore file in case the frontend module is not enabled. See https://github.com/adobe/aem-project-archetype/pull/296/files#diff-346cccad6cc3468e44003d42921faec1R29

Reply:
@bpauli yes #278. There was a second PR that I merged for this that only adds the ignore statements if the ui.frontend is enabled https://github.com/adobe/aem-project-archetype/commit/291ad7306fa96743b8e6eeea083c8d2c83a8c033